src/main/java/dynamo/RidesDao.java

```diff
@@ -25,5 +35,52 @@ public RidesDao() {
 
 
 
+	// inserts a new person into the table with an email & a name
+	public void insert(String email, String name, int year, String phoneNumber, String church, boolean attendance) {
+
+		Table table = dynamoDB.getTable("RideTable");
+
+		final Map<String, Object> infoMap = new HashMap<String, Object>();
+		infoMap.put("Email", email);
+		infoMap.put("Name", name);
+		infoMap.put("Church", church);
+		infoMap.put("year", year);
+		infoMap.put("Phone Number", phoneNumber);
+		if (attendance) { // attends or not
+			infoMap.put("attendance", "yes");
+		} else {
+			infoMap.put("attendance", "no");
+		}
 
+		try {
+			System.out.println("Adding a new item...");
+			PutItemOutcome outcome = table
+					.putItem(new Item().withPrimaryKey("Email", email, "Name", name).withMap("info", infoMap));
+
+			System.out.println("PutItem succeeded:\n" + outcome.getPutItemResult());
+
+		} catch (Exception e) {
+			System.err.println("Unable to add " + name + " with email: " + email);
+			System.err.println(e.getMessage());
+		}
+	}
+
+	//updates table based on email & primary key
+	public void update(String newEmail, String name, String oldEmail) {
```

Comment:
This function is a little more complex than it seems.
Given an email, we need to be able to update all other table attributes. We need to be able to accept any amount of new information and update it while keeping old information the same.

For example, if I make an entry with [email: jason@jason.com, Name: Jason, PhoneNumber: 1111111111, blah blah]

I should be able to call update with [email: jason@jason.com and PhoneNumber: 2222222222] and update just the phone number without altering the other fields